### ISSUE TITLE
ODN/UnifiedViews: Unexpected error occurred in scheduled task: Config is missing property: exec.log.history

### ISSUE THREAD

**Issue by hanecak** [OpenDataNode/open-data-node#264]
This just tracks upstream issue: https://github.com/UnifiedViews/Core/issues/585

regarding:

```
2016-06-20 00:00:00,001 [taskScheduler-3] ERROR exec: dpu: o.s.s.support.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task.
cz.cuni.mff.xrg.odcs.commons.app.conf.MissingConfigPropertyException: Config is missing property: exec.log.history.
        at cz.cuni.mff.xrg.odcs.commons.app.conf.AppConfig.getString(AppConfig.java:143) ~[commons-app-2.2.0.jar:na]
        at cz.cuni.mff.xrg.odcs.commons.app.conf.AppConfig.getInteger(AppConfig.java:155) ~[commons-app-2.2.0.jar:na]
        at cz.cuni.mff.xrg.odcs.backend.pruning.Log.execute(Log.java:56) ~[backend-2.2.0.jar:na]
        at sun.reflect.GeneratedMethodAccessor71.invoke(Unknown Source) ~[na:na]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.7.0_101]
        at java.lang.reflect.Method.invoke(Method.java:606) ~[na:1.7.0_101]
        at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:64) ~[spring-context-3.1.4.RELEASE.jar:3.1.4.RELEASE]
        at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:53) ~[spring-context-3.1.4.RELEASE.jar:3.1.4.RELEASE]
        at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:81) [spring-context-3.1.4.RELEASE.jar:3.1.4.RELEASE]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471) [na:1.7.0_101]
        at java.util.concurrent.FutureTask.run(FutureTask.java:262) [na:1.7.0_101]
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:178) [na:1.7.0_101]
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:292) [na:1.7.0_101]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) [na:1.7.0_101]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) [na:1.7.0_101]

```

**Comment by hanecak** [comment]
Fix merged in upstream to 'develop' branch, now waiting for the fix to occur in stable release.

**Comment by hanecak** [comment]
Fixed in UnifiedViews 2.3.2 which is included in ODN 1.6.0 .
